app/top-page.module.css

```
@@ -137,69 +137,80 @@
   color: rgb(255, 255, 255);
   margin-left: 5%;
+  margin-top: 10px;
 }
 .newsList {
+  font-size: 1.1rem;
+  color: rgb(255, 255, 255) !important;
+  margin-left: 10px;
+  margin-bottom: 5px;
   list-style: none;
   padding: 0 5%;
   margin: 0;
```

Stylelint (17.12.0)

[error] Overridden property "margin-bottom" by shorthand "margin" (declaration-block-no-shorthand-property-overrides)

(declaration-block-no-shorthand-property-overrides)